### ISSUE TITLE
add function to shorten decimals

### ISSUE THREAD

**Issue by bnidevs** [bnidevs/the-hotbox#23]
given a var `f` which is a float, shorten the string representation of `f` to 2 decimal places

so the function signature should be

```
var shorten_float = (f) => {
//
//   do something
//
return shortened_string;
}
```

**Comment by ilmanzo** [comment]
Hello, glad to contribute! Can I take this issue ?

**Comment by bnidevs** [comment]
@ilmanzo not taking any outside contributions at the moment sorry

**Comment by bnidevs** [comment]
Closed #31
